aptos-move/framework/supra-framework/sources/automation_registry.move

```diff
@@ -0,0 +1,89 @@
+/// Supra Automation Registry
+///
```

Comment:
`public fun get_active_tasks() : vector<AutomationMetaData>`

`on_new_epoch()` should perform clean up and updation of state

Do we want to restrict only for entry function or should we allow scripts as well?